src/processes.c

```diff
@@ -230,7 +232,7 @@ reuse_or_add_context (struct ipc_context *ctx)
  * FORKFAILED
  */
 pid_t
-create_ipc_process (ipc_process_func func, void *args)
+create_ipc_process (kb_t main_kb, ipc_process_func func, void *args)
```

Comment:
if only the global set variable is used for checking kb inconsistencies, what is the passed main kb used for?